cms/models.py

```diff
@@ -352,6 +355,10 @@ def product(self):
 
     template = "product_page.html"
 
+    def get_admin_display_title(self): 
+        """Gets the title of the course in the speacified format"""
+        return f"{self.course.readable_id} | {self.course.title}"
```

Comment:
This thing is already being done in the course object str formatting, https://github.com/mitodl/mitxonline/blob/main/courses/models.py#L133. I think we can reuse that.